demosplan/DemosPlanCoreBundle/ResourceTypes/UserResourceType.php

```diff
@@ -41,10 +48,13 @@
  * @property-read DepartmentResourceType $department
  * @property-read RoleResourceType $roles
  */
-final class UserResourceType extends DplanResourceType implements UserResourceTypeInterface
+final class UserResourceType extends DplanResourceType implements UserResourceTypeInterface, ReadableEsResourceTypeInterface
```

Comment:
- I think the correct ResourceType is actually AdministratableUser
- Check this line: config/services.yml:404 -> the AdministratableUser is used for the JsonApiEsService
- I guess the route needs to be adjusted to use adminsitrableUser